tripper/triplestore.py

```diff
@@ -168,7 +168,7 @@ def serialize(
         self._check_method("serialize")
         return self.backend.serialize(destination=destination, format=format, **kwargs)
 
-    def query(self, query_object, **kwargs) -> "List[Tuple[str, ...]]":
+    def query(self, query_object, **kwargs) -> "List[Triple]":
```

Comment:
This is actually not correct. The return value of `query()` depends on the sparql query. It just returns a list of tuples for each match. The tuple is not a triple but contains as many fields as the query asks for.

Reply:
so the original return value is correct?